runtime/src/bank.rs

```diff
@@ -1743,6 +1755,45 @@ mod tests {
         assert_eq!(bank1.block_height(), 1);
     }
 
+    impl Bank {
```

Comment:
nice

Reply:
A few more comments here would be awesome :)

Reply:
Thanks for the feedback! Piece of cake once I fix the snapshot issue. :)